src/Components/Modal.js

```diff
@@ -3,127 +3,156 @@ import styled from 'styled-components';
 // import { mdiWikipedia } from '@mdi/js';
 import {Icon} from '@iconify/react';
 
-export default function Modal({open, onClose, content}) {
+export default function Modal({isOpen, onClose, content}) {
   console.log(content);
   return (
-    open && (
+    isOpen && (
       <>
-        <Styled_Close_Button type="button" onClick={onClose}>
-          <Icon
-            icon="ant-design:close-square-outlined"
-            width="45"
-            height="45"
-          />
-        </Styled_Close_Button>
-        <StyledModalOuter>
-          <StyledModalInner>
-            <Styled_Wikipedia_Icon>
+        <Styled_Outer_Modal>
```

Comment:
Please think about the naming of your components.

Reply:
I will thank you